tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,29 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self):
+    return self._visualize_dataset(self._builder, 'train')
+
+  def _visualize_dataset(self, builder, split):
+    """Visualize dataset statistics with FACETS DIVE."""
+    filetype_suffix = builder._file_format_adapter.filetype_suffix  # pylint: disable=protected-access
+    if filetype_suffix not in ["tfrecord", "csv"]:
+      raise ValueError(
+        "Cannot generate statistics for filetype {}".format(filetype_suffix))
+    filepattern = naming.filepattern_for_dataset_split(
+      builder.name, split, builder.data_dir, filetype_suffix)
+    if filetype_suffix == "csv":
+      statistics = tfds.core.lazy_imports.tfdv.generate_statistics_from_csv(
+        filepattern)
+    else:
+      statistics = tfds.core.lazy_imports.tfdv.generate_statistics_from_tfrecord(
+        filepattern)
+    logger = logging.getLogger()
+    logger.disabled = True
```

Comment:
You should add a comment on why this is necessary.

Reply:
Because tfdv has a lot of warning so I needed to close.

Reply:
Sounds good, but in this case, I would add this as a kwargs in case the user still want to access the logs, something like:

`info.visualize_dataset(disable_logging=True)`

Reply:
Sorry about that. `tfdv.visualize_statistics` hasn't got warnings, these warnings are given by `generate_statistics_from_tfrecord`. And I think this warnings are unnecessary. Check this [log](https://gist.github.com/us/e6ec3ece40a250b5231fdda096ed49dc).